spec/integration/posts_index_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+# integration test for posts index
+describe 'Posts Index', type: :system do
+  let!(:author) { User.create(name: 'Akai321', photo: 'https://Akaiiii.jpg', bio: 'Martial Artist') }
+  let!(:user) { User.create(name: 'Kenshiro', photo: 'https://Kenshiro.jpg', bio: 'Martial Artist') }
+  let!(:post1) do
+    Post.create(
+      author:,
+      title: 'The way of the Water',
+      text: 'In martial arts, the way of the water is the way of the soft and yielding.'
+    )
+  end
+  let!(:post2) do
+    Post.create(
+      author:,
+      title: 'The way of the Fire',
+      text: 'In martial arts, the way of the fire is the way of the hard and unyielding.'
+    )
+  end
```

Comment:
Good job, writing some of the required integrations, however, there are many more you've skipped. I recommend that you implement these specs in order to satisfy the milestone's requirement:

- Tests that asserts you can see the user's profile picture.
- Tests that asserts you can see the user's username.
- Tests that asserts you can see a post's title.
- Tests that asserts you can see some of the post's body.
- Tests that asserts you can see the first comments on a post.
- Tests that asserts you can see how many comments a post has.
- Tests that asserts you can see how many likes a post has.
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.

Reply:
- [ ] Good job in adding the tests required by the previous reviewer, but you missed a  `Tests that asserts you can see how many likes a post has`. kindly fix it